### DISCUSSION TITLE
ONVIF Device Test Tool - TLS Tests Issue

### DISCUSSION THREAD

**Discussion by FNS-ONVIF** [onvif/testspecs#23] [Q&A] [answered]
In our lab we have several computers to run the Onvif Test Tool on, but only one of them runs the HTTPS tests correctly, with the rest of the computers, these tests fail consistently.

Therefore, we would like to know, if it is necessary to have any special configuration either in the computers or in the ONVIF Test Tool, so that these tests can be executed without any problem, since only being able to perform these tests on a single computer, blocks us quite a lot when testing with several cameras at the same time.

Thanks in advance!

**Comment by davidmneedham** [comment, marked as answer]
No special configuration is necessary. Compare configuration of the functioning machine with the non-functioning machines.
